.github/workflows/tag.yml

```diff
@@ -2,10 +2,13 @@ name: Auto-Tag
 
 on:
   workflow_run:
-    workflows: [build]
+    workflows: ["CI"]
     types:
       - completed
 
+  push:
+    tags: ['v[0-9]+.[0-9]+.[0-9]+']
```

Comment:
Will this catch `beta` suffixes, like:`v1.1.1-beta`

Reply:
Oooh good point. Reverting this to v* as I'm not confident enough in the pattern above now..